docs/github/v2/01-getting-started.md

```diff
@@ -75,7 +75,9 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: Library
-          key: Library
+          key: Library-${{ hashFiles('Assets/**', 'Packages/**', 'ProjectSettings/**') }}
```

Comment:
I asked myself if it was worth mentioning that this only works if the project is at the root of the repository, but it's already the case on line 56 👍 This looks perfectly fine to me